src/js/components/Meter/__tests__/__snapshots__/Meter-test.tsx.snap

```diff
@@ -249,6 +256,7 @@ exports[`Meter boundary values 1`] = `
   class="c0"
 >
   <svg
+    aria-label="circle meter with value 2.26% out of 10.299999999999999"
```

Comment:
@jcfilben still not sure about the wording.. what you think

Reply:
okay I wonder if we take out the `%` and just have the user pass in whatever units they want

Reply:
Yeah I think we should take out the `%`

Reply:
agree took out in last commit